### ISSUE TITLE
[Interview] Book Launch interview "Building Green Software"

### ISSUE THREAD

**Issue by ursvill** [Green-Software-Foundation/writers#109]

**Comment by ursvill** [comment]
Book: [Building Green Software](https://www.oreilly.com/library/view/building-green-software/9781098150617/)
By Anne Currie, Sarah Hsu, Sara Bergman

**Comment by ursvill** [comment]
Draft Interview Questions - @NAMRATA-WOKE please comment/add

1. What compelled you to write the book?
2. Why does the world need green software?
3. Who should read the book?
4. Can you give me a nugget or two of what's in the book?
5. What are the most important aspects of green software for you personally?

**Comment by NAMRATA-WOKE** [comment]
@ursvill instead of why does the world need green software, I recommend we ask:

What's new in this book that "veterans" in the space can look forward to seeing? Or something like..

How does this book approach the narrative of green software? 

I'd also like to ask them:

What was your experience like collaborating on this book? What surprised you?

**Comment by seanmcilroy29** [comment]
@NAMRATA-WOKE - Sarah's email address - Sara.Bergman@microsoft.com

**Comment by ursvill** [comment]
Awesome, thanks!

**Comment by jawache** [comment]
@NAMRATA-WOKE / @seanmcilroy29 let's delete those messages with peeps emails, the repo is public 😄

**Comment by ursvill** [comment]
![Image](https://github.com/Green-Software-Foundation/writers/assets/114167830/d5a07d29-dfcf-4a93-a9e2-7110d6637108)

**Comment by ursvill** [comment]
Hey Urs,
I'm not sure how long answers you want, I provided pretty short ones, let me know if you need more :) 
I've attached one photo, I think Anne might have another one, I could not find it at least.

**What compelled you to write the book?**
Doesn't everyone want to write a book at some point in their life? Jokes aside, having the ability to do it with Anne and Sarah had a lot to do with it. We have worked with the GSF for 2 years and know each other and we are aligned in the message around green software. Even though we had never met all three in person when we signed! There was also a gap in the market, a lot of people are asking more and more in-dept questions about green software, as frequent public speakers this was something we noticed (as I'm sure anyone in the space has). A book gives us the gives us the ability for truly in-dept answers in a way that a ~45-minute conference talk does not.

**Why does the world need green software?**
We need green everything. If we are to reach the Paris agreement goal, we cannot afford to NOT make software (a rapidly growing sector) green.

**Who should read the book?**
Anyone in the software space! Some parts truly get into the nitty gritty, others are more of an overview, so there will be something for everyone. From the new software engineer to CTOs. But really, we won't gatekeep, if you think it sounds interesting, we are glad to have you!

**What's new in this book that "veterans" in the space can look forward to seeing?**
While we aim to provide a high-level introduction to the field, we will go deep into the technical details of many of the architectural approaches which are required for green software. We will also cover some of the things you may kind-of-sort-of know about but never really invested the time to look into. For example, how will renewables affect the grid? What really is the difference between carbon neutral and net zero? How do the different cloud provider's carbon measurements tools compare to each other? 

**Can you give me a nugget or two of what's in the book?**
Not sure how much you want here Urs, are you looking for a quote from the book or more of a content description?

**What are the most important aspects of green software for you personally?**
As a software engineer, it is the actionability for me. Galvanizing people, bringing climate action into the workspace on a level that doesn't only touch the company leadership, that appeals to me. I think most people want to do something, anything, to help be part of the climate solution but are lacking the awareness or tools. 

**What was your experience like collaborating on this book? What surprised you?**
So far, lots of fun! I'm glad we get to do it together, getting feedback from Anne and Sarah definitely makes anything I write better. Having different perspectives also helps makes the content more diverse and helps us find any "blind spots". But we are not done yet, so ask us again in a few months and we'll see 😉 What surprised me the most has been the support of people outside our "bubble"! People has been very positive and supportive of the book and offered help and words of encouragement.

//Sara

**Comment by NAMRATA-WOKE** [comment]
Need to push the date back because Anne needs more time to respond to Urs. Hopefully we can get this out in August or Early September.

**Comment by ursvill** [comment]
not hearing back from Anne; asked Sara to expand on her answers

**Comment by NAMRATA-WOKE** [comment]
@ursvill still waiting on Anne to respond and provide more context for the article. @jawache to ping Anne for support. @ursvill will follow-up on Tuesday next week.

**Comment by ursvill** [comment]
I think we should postpone until closer to the actual book launch.

**Comment by NAMRATA-WOKE** [comment]
Postpone till December.

**Comment by NAMRATA-WOKE** [comment]
@ursvill https://gotopia.tech/episodes/243/building-green-software
